mlir/lib/Dialect/AMDGPU/Transforms/TransferReadToLoad.cpp

```diff
@@ -141,6 +243,7 @@ struct TransferReadLowering final : OpRewritePattern<vector::TransferReadOp> {
 void mlir::amdgpu::populateAmdgpuTransferReadToLoadPatterns(
     RewritePatternSet &patterns) {
   patterns.add<TransferReadLowering>(patterns.getContext());
+  vector::populateVectorTransferLoweringPatterns(patterns);
```

Comment:
I'm not sure we need this here? People can populate that on their own?

Reply:
I think these sorts of limited-applicability patterns don't often populate their next step - I say keep the attribute

Reply:
Sure, will get rid of this rewrite pattern.